packages/project-management-automation/lib/if-not-fork.js

```diff
@@ -8,14 +13,17 @@
  * @return {Function} Enhanced task.
  */
 function ifNotFork( handler ) {
-	return ( payload, ...args ) => {
+	const newHandler = ( payload, ...args ) => {
 		if (
 			payload.pull_request.head.repo.full_name ===
 			payload.pull_request.base.repo.full_name
 		) {
 			return handler( payload, ...args );
 		}
+		debug( `main: Skipping ${ handler.name } because we are in a fork.` );
 	};
+	newHandler.name = handler.name;
```

Comment:
I don't think this works. It appears you cannot reassign the name of a function after it has been created.

https://2ality.com/2015/09/function-names-es6.html#caveat-names-assigned-at-creation

```js
const foo = () => {};
foo.name = 'bar';
console.log( foo.name );
// "foo"
```

The log still shows without the names: https://github.com/WordPress/gutenberg/pull/20034/checks?check_run_id=426111782

Possible alternatives:

- https://stackoverflow.com/a/33067824
- Change the implementation to stop reading from the function name, but instead from some explicit (if albeit redundant) `name` property value of the [entry itself](https://github.com/WordPress/gutenberg/blob/aae6d1bf010707689356181662b82b83b09fc59c/packages/project-management-automation/lib/index.js#L17-L21)

Reply:
Done!